README.md

```diff
@@ -1,21 +1,41 @@
 # kubevirt-cpu-nfd-plugin
 
-**kubevirt-cpu-nfd-plugin** is plugin for [kubevirt-node-labeller](https://github.com/ksimon1/kubevirt-cpu-node-labeller). It creates list of all supported cpu models on host, which cpu-node-labeller then exposes as node labels.
-
-**Usage:**
-```
-oc apply -f cpu-model-labeller-plugin.yaml
-```
+**kubevirt-cpu-nfd-plugin** is plugin for [kubevirt-node-labeller](https://github.com/ksimon1/kubevirt-cpu-node-labeller). It creates list of all supported cpu models and features on host, which cpu-node-labeller then exposes as node labels.
 
 **How it works**
 
-In the libvirt container, output of virsh domcapabilities is saved into `/host-hook/virsh_domcapabilities.xml` folder. This folder is shared between libvirt and cpu-node-labeller container. Plugin parses libvirt data and prints data to stdout in format:
+Plugin parses libvirt data (cpu models, features) and prints data to stdout in format:
 ```
 /cpu-model-haswell
 /cpu-model-core2duo
+/cpu-feature-aes
 ...
 ```
-cpu-node-labeller takes this output and creates labels from them.
+cpu-node-labeller takes this output and creates labels from them. 
+
+The plugin can hide old cpus. This can be done by creating config map:
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubevirt-cpu-plugin-configmap
+data:
+  cpu-plugin-configmap.yaml: |- 
+    obsoleteCPUs:
```

Comment:
Isn't this also handled by the minCPU implicitly?

Reply:
issue is, that virsh domcapabilities doesn't order cpu models in correct order (e.g. from oldest to newest), so i cannot say which model is older than minCPU model. Maybe there is a way how can libvirt show models in correct order?

Reply:
Why not just check whether the flags are a subset of the minimal cpu?

Reply:
Some really old models(486, pentium3 contain vme, core2due contains monitor) contains features, which e.g. Conroe, Penryn and newer don't have, so these old models are not subsets of newer models. But it can be solved by hardcoded features, which should not be in the list.